dig.go

```diff
@@ -111,6 +111,20 @@ type InvokeOption interface {
 	unimplemented()
 }
 
+// A graphNode represents a node in the dependency DOT graph
+type graphNode struct {
```

Comment:
Seeing as "graph" is a fairly overloaded term here, let's consider naming this `dotNode` (since this is represented in DOT-format).